src/controllers/VaultController.ts

```diff
@@ -19,9 +20,10 @@ export class VaultController {
   constructor(opts: Options) {
     this.store = new ObservableStore(
       Object.assign({}, { locked: null, initialized: null }, opts.initState, {
-        locked: true,
+        locked: !opts.password,
       })
     );
+    this.password = opts.password;
```

Comment:
It seems `this.password` isn't read anywhere. @smelnikov any idea why it's here?

Reply:
Yeah. Looks like an outdated code snippet.
The original idea was for wallet/identity controllers to look at this field to unlock their encrypted store values.
This was later resolved with dependency injection of these controllers.

```javascript
unlock(password) {
    this.wallet.unlock(password);
    this.identity.unlock(password);

    this.password = password;
    this.locked = false;
  }
  ```